applications/doodle/src/module/cf_sms_user.erl

```diff
@@ -25,7 +25,10 @@
 -spec handle(wh_json:object(), whapps_call:call()) -> 'ok'.
 handle(Data, Call1) ->
     UserId = wh_json:get_ne_value(<<"id">>, Data),
-    Call = doodle_util:set_callee_id(UserId, Call1),
+    Funs = [{fun doodle_util:set_callee_id/2, UserId}
+            ,{fun whapps_call:kvs_store/3, <<"target_owner_id">>, UserId}
```

Comment:
adding target_owner_id + view change should resolve https://github.com/2600hz/kazoo/pull/952 but still worried about above (line 28)